R/get_source_expressions.R

```diff
@@ -484,22 +484,26 @@ get_source_expression <- function(source_expression, error = identity) {
   parse_error <- FALSE
 
   tryCatch(
-    source_expression$parsed_content <- parse(
-      text = source_expression$content,
-      srcfile = source_expression,
-      keep.source = TRUE
-    ),
+    {
+      source_expression$parsed_content <- parse(
```

Comment:
I think more readable / canonical is to avoid the double assignment altogether, e.g.

```r
parsed_content <- tryCatch(...)
if (inherits(parsed_content, ...)) { ... }

...

source_expression$parsed_content <- parsed_content
```

Reply:
Good suggestion. 

I have done this, but I couldn't use the suggested name (`parsed_content`) for the reason mentioned in my comment.

If I am mistaken, let me know.